Vladimir_Mustafa/HW9/Task 1.py

```diff
@@ -0,0 +1,26 @@
+import random
```

Comment:
good job
look at this code
import random

num = random.randint(1, 100)
max_attempts = 10

print('Guess a number from 1 to 100. You have 10 attempts.')

for attempt in range(1, max_attempts + 1):
    user_num = int(input(f'Attempt {attempt}: Enter a number: '))

    if 1 <= user_num <= 100:
        if user_num < num:
            print('The number is higher. Try again.')
        elif user_num > num:
            print('The number is lower. Try again.')
        else:
            print(f'Congratulations, you win! The number is {num}')
            break
    else:
        print('Please enter a valid number between 1 and 100.')

if attempt == max_attempts and user_num != num:
    print(f'Sorry, you have used all {max_attempts} attempts. The correct number was {num}.')